Adyen/Helpers/ImageLoader/UIImageView+ImageLoading.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+@_spi(AdyenInternal)
+public extension UIImageView {
+    @discardableResult
+    func load(url: URL, using imageLoader: ImageLoading, placeholder: UIImage? = nil) -> AdyenCancellable {
+        imageLoader.load(url: url) { [weak self] image in
+            guard let image else {
+                self?.image = placeholder
```

Comment:
This feels like a part of business-logic actually. 
I can imagine we might need to show a placeholder before loading starts and update it with the loaded image after. However, I am not aware of real business logic but this just seem to be capturing it which is not ideal for this level of abstraction.